src/ui/public/state_management/monitor_changes.js

```diff
@@ -0,0 +1,22 @@
+function monitorStateChanges(state, handlerFn, cleanupFn) {
```

Comment:
Not according to the [style guide](https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#use-lowercamelcase-for-variables-properties-and-function-names)